tools/core-build-tasks/src/tasks/copy.ts

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) Mojang AB.  All rights reserved.
+
+import { FileSystem, JsonFile } from "@rushstack/node-core-library";
+import path, * as Path from "path";
+import { getOrThrowFromProcess } from "./helpers/getOrThrowFromProcess";
+import { getGameDeploymentRootPaths } from "./helpers/getGameDeploymentRootPaths";
+import { MinecraftProduct } from "../platforms/MinecraftProduct";
+
+/**
+ * For each output path, this is the configuration for the files to copy.
+ * Folders will be copied to the output folder entirely.
+ */
+export type FilesToCopy = { files?: string[]; folders?: string[] };
+
+/**
+ * Format for configuration files. There are sets of files to copy for each output path specified.
+ *
+ * These should be a scheme for validation purposes
+ */
+export type ConfigurationFormat = {
+  [outputPath: string]: FilesToCopy | FilesToCopy[];
+};
```

Comment:
For the copy task, can we kill the configuration entirely and either have this copy task be more specific, i.e. `copyToBehaviorPacks`, or have the paths passed in via the `CopyTaskParameters` so its just in the just.config.js instead.

The complexity of copy is way lower for the samples scenario, so if we can get away without a configuration file, I think that is way preferable. This needs to be as plug and play as possible.

Reply:
Simplified, using collections in the input object.